src/lib/components/side-nav.svelte

```diff
@@ -96,6 +106,25 @@
         <div class="nav-title">{translate('feedback')}</div>
       </NavRow>
     </slot>
+    <NavRow {isCloud} handleClick={() => ($labsMode = !$labsMode)}>
```

Comment:
Looks like having the `<li/>` and `<button/>` element (and both with the nav-row class) is creating some wonkiness 🤔 

![Screenshot 2023-08-10 at 11 27 24 AM](https://github.com/temporalio/ui/assets/15069288/ba50cd60-f932-48b1-820d-05512e83e851)
![Screenshot 2023-08-10 at 11 20 56 AM](https://github.com/temporalio/ui/assets/15069288/05ad09f7-8e12-4feb-ac9f-cd42c210d19f)